windows_container_workloads/enabling-windows-container-workloads.adoc

```diff
@@ -0,0 +1,35 @@
+[id="enabling-windows-container-workloads"]
+= Enabling Windows container workloads
+include::modules/common-attributes.adoc[]
+:context: enabling-windows-container-workloads
+
+toc::[]
+
+Before adding Windows workloads to your cluster, you must install the Windows Machine Config Operator (WMCO), which is available in the {product-title} OperatorHub. The WMCO orchestrates the process of deploying and managing Windows workloads on a cluster.
+
+[discrete]
+== Prerequisites
```

Comment:
@codyhoag @Anandnatraj Given https://github.com/openshift/kubernetes/pull/446, I wounder if we should call out the exact 4.6.z that PR is in as a minimum version requirement?

Reply:
Good idea to call it out...will help lower support tickets if customers are trying the WMCO on an older/unsupported 4.6.z cluster

Reply:
4.6.8 should be the minimum version then.

Reply:
I will add that as a prereq; thanks!